app/src/components/integrations/elements/integrationSettings/connectionSection/connectionSection.jsx

```diff
@@ -206,6 +210,7 @@ export class ConnectionSection extends Component {
               <div
                 className={cx('connection-block', {
                   'connection-block-failed': !connected,
+                  'connection-block-disabled': blocked && availableProjectIntegrations.length > 0,
                 })}
```

Comment:
The same thing.
It is important to use length>0 only when we use conditional rendering.
It is not mandatory to fix.

Reply:
without this, styling will be improper